packages/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -129,12 +129,12 @@ export interface DataGridPropsWithDefaultValues {
   checkboxSelectionVisibleOnly: boolean;
   /**
    * Column region in pixels to render before/after the viewport
-   * @default 100
+   * @default 150
    */
   columnBufferPx: number;
   /**
    * Row region in pixels to render before/after the viewport
-   * @default 100
+   * @default 150
```

Comment:
Should we initialize it based on the `props.rowHeight`, maybe `props.rowHeight * 3` or something?
I want to [make](https://github.com/mui/mui-x/pull/12317/files#diff-237b220ab18e9e327a1bdd56eba704ca2f7fbb51b48b57c7973025827b17fc15R22-R24) the initial props' object a function in the datasource feature (a potential BC), which could be helpful in deducing initial props based on other props, we might use it to initialize `rowBufferPx`, wdyt?

Reply:
We'd need to set the default value to `undefined`, then we'd be able to pick the best value when the logic runs. Does that sound good?

Reply:
I'll leave it as it is for now, changing the default value later is not a breaking change I think.

Reply:
Sound good 👍